css/style.css

```diff
@@ -1,4 +1,441 @@
 .body {
   font-family: 'Montserrat', sans-serif;
   font-weight: normal;
+  max-width: 1440px;
+  min-width: 320px;
+  overflow-x: hidden;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  box-sizing: border-box; 
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)